hera_cal/omni.py

```diff
@@ -1297,7 +1300,10 @@ def omni_apply(files, opts):
             for bl, k in zip(*np.unique(mir.baseline_array, return_index=True)):
                 blmask = np.where(mir.baseline_array == bl)[0]
                 ai, aj = mir.baseline_to_antnums(bl)
-                if not ai in cal.ant_array or not aj in cal.ant_array:
+                if ai not in cal.ant_array or aj not in cal.ant_array:
```

Comment:
I think you'll need to move this block of code into the next for loop because nsp doesn't exist yet.

Reply:
Ah, good catch. I'll move it